aten/src/ATen/native/Activation.cpp

```diff
@@ -722,7 +722,22 @@ Tensor softshrink_backward(const Tensor & grad, const Tensor & self, const Scala
   return iter.output();
 }
 
+bool use_mkldnn(const Tensor& input) {
+#if AT_MKLDNN_ENABLED()
+  if (!at::globalContext().userEnabledMkldnn()) {
+    return false;
+  }
+  return input.options().backend() == at::Backend::CPU &&
```

Comment:
We can merge `mkldnn_gelu` and ` mkldnn_gelu_cpu` into one function to simply the code.

First:
```C++
return (input.is_mkldnn()) || // input is mkldnn Tensor
    (input.device().is_cpu() && input.scalar_type() == kBFloat16)`; // input is dense layout and bfloat16
```